tests/scripts/release-post.test.js

```diff
@@ -1,32 +1,31 @@
 'use strict';
 
-const test = require('tape');
+const test = require('node:test');
+const assert = require('node:assert/strict');
 const nock = require('nock');
 const proxyquire = require('proxyquire').noCallThru();
 const path = require('path');
 
-test('explicitVersion(<version>)', (t) => {
+test('explicitVersion(<version>)', async (t) => {
   const releasePost = require('../../scripts/release-post');
 
-  t.test('resolves when given a string argument', (t) => {
+  await t.test('resolves when given a string argument', (_, done) => {
     releasePost.explicitVersion('4.1.1').then((version) => {
-      t.equal(version, '4.1.1');
-      t.end();
+      assert.equal(version, '4.1.1');
+      done();
     });
   });
 
-  t.test('rejects when given an falsy argument', (t) => {
+  await t.test('rejects when given an falsy argument', (_, done) => {
```

Comment:
I would stick with one style of async programming 
async / await, .then, or callbacks. Not all 3 at once.

Reply:
@benhalverson: Thank you and it's been fixed.